src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -650,7 +650,7 @@ internal override void Dispose(bool isDisposing)
                 {
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 10, but got 9.39843271760412.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1959
1959:             $pathologicalRatio | Should -BeGreaterThan 10
```